### DISCUSSION TITLE
Test d'application PRO

### DISCUSSION THREAD

**Discussion by CI4CB** [WenesLimem/PRO#73] [Q&A]
Bonjour,

Je teste votre application et j'aimerais savoir comment puis-je modifier le menu disponible. Est-ce possible? Avec quels droits d'accès?

Merci.

CPN.

**Comment by mascot27** [comment]
Bonjour,
Si vous parlez de la sidebar de navigation alors, soit vous avez accès à la vue "employee" soit la vue guest en utilisateur non-authentifié.

Si vous parlez des recettes affichées, celles-ci sont directement celle que l'on retrouve dans la partie recette. Les catégories sont affichées en tant qu’onglet dans le menu et les recettes sont éditables sans soucis depuis l'utilisateur Employee.

Tenez-nous au courant si cela répond à votre question.

@CI4CB

**Comment by CI4CB** [comment]
Bonjour,

Je parlais des “plats” proposés dans le menu… est-ce que une “recette” fait automatiquement partie du menu? Ce n’est pas clairement établi.

Merci et bonne journée,

—
Carlos Peña




Le 28 juin 2021 à 18:08, Corentin ZELLER ***@***.******@***.***>> a écrit :



Bonjour,
Si vous parlez de la sidebar de navigation alors, soit vous avez accès à la vue "employee" soit la vue guest en utilisateur non-authentifié.

Si vous parlez des recettes affichées, celles-ci sont directement celle que l'on retrouve dans la partie recette. Les catégories sont affichées en tant qu’onglet dans le menu et les recettes sont éditables sans soucis depuis l'utilisateur Employee.

Tenez-nous au courant si cela répond à votre question.

**Reply by mascot27** [reply]
Exactement, le tout est lié. Une recette sera affiché automatiquement dans le menu.
